libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorRecognizer.cs

```diff
@@ -63,30 +63,12 @@ public OrchestratorRecognizer([CallerFilePath] string callerPath = "", [CallerLi
         /// </summary>
         /// <param name="modelFolder">Specifies the base model folder.</param>
         /// <param name="snapshotFile">Specifies full path to the snapshot file.</param>
-        /// <param name="resolver">Label resolver.</param>
-        public OrchestratorRecognizer(string modelFolder, string snapshotFile, ILabelResolver resolver = null)
+        /// <param name="resolverMockup">Label resolver mockup.</param>
+        public OrchestratorRecognizer(string modelFolder, string snapshotFile, ILabelResolver resolverMockup = null)
```

Comment:
Is this constructor only for tests? Asking because of the mockup variable name

Reply:
I was about to renamed it to resolverExternal.

Reply:
Given that this shipped as public in our surface area, a name without the mockup word will be better, i.e. resolver. And then you cna rename `_isResolverMockup` to `_isResolverInjected` if this variable is still needed. Mostly because since it is public, developers could pass in a legitimate label resolver and not a mockup so the name would be misleading